test/ranges-test.cc

```diff
@@ -420,6 +423,10 @@ TEST(ranges_test, join_tuple) {
   auto t4 = std::tuple<float>(4.0f);
   EXPECT_EQ(fmt::format("{}", fmt::join(t4, "/")), "4");
 
+  // Tuple-like.
+  auto t5 = tuple_like{42, "foo"};
+  EXPECT_EQ(fmt::format("{}", t5), "(42, \"foo\")");
```

Comment:
This test case looks identical to one of the above:

```
  auto t = tuple_like{42, "foo"};
  EXPECT_EQ(fmt::format("{}", t), "(42, \"foo\")");
```

Did you mean to use `join` here?

Reply:
Yes, I lost ``fmt::join``.